### ISSUE TITLE
Establish an external advisory group

### ISSUE THREAD

**Issue by kelle** [astropy/project#68]
It would be nice to have a group of senior folks who comprise an external advisory group. These folks should be familiar with the project, have experience with other large software projects, and able to give us advice on management, funding, workflows. They are folks which we might email as needed but would also meet with annually to discuss our accomplishment, roadmaps, and challenges.

**Comment by bsipocz** [comment]
Glad to see that ideas that came up before, now getting accepted as options.

**Comment by pllim** [comment]
Has this been superseded by https://github.com/astropy/astropy.github.com/pull/584 ?

**Comment by astrofrog** [comment]
I don't think so, the strategic role would be mostly internal to the project?

**Comment by pllim** [comment]
Do we need so many "advisory/strategic" groups internal and external? I feel like that is overkill.

**Comment by kelle** [comment]
We have zero external advisory groups at the moment and are in the verge of having one internal role. If you think one each is overkill, that's fine. But I think we desperately need the internal one and we'll be a better project if we have the external one.

**Comment by pllim** [comment]
Does Astropy Strategic Planning Committee count?
